app/sidekiq/publish_translation_job.rb

```diff
@@ -0,0 +1,9 @@
+class PublishTranslationJob
+  include Sidekiq::Job
+  queue_as :default
```

Comment:
The Sidekiq syntax we tend to use is `sidekiq_options queue: ...`, but it's not necessary to take the default.

Reply:
@knutsenm we can implement then like this next

```ruby
  sidekiq_options queue: :publish_translation
```

what do you think?

Reply:
There will likely not be a need for multiple queues, so the default should suffice; @frett ?